app/descriptor/views.py

```diff
@@ -0,0 +1,8 @@
+from flask import render_template
+from flask.ext.login import login_required
+from . import descriptor
+
+@descriptor.route('/')
+@login_required
+def view():
```

Comment:
Yeah, I've thought about it too, but it seems like a bad style to have more than one index. What do you think about that?